gpkitmodels/GP/aircraft/wing/gustloading.py

```diff
@@ -2,32 +2,54 @@
 import os
 from numpy import pi, hstack, cos
 import pandas as pd
-from gpkit import Variable, Vectorize
+from gpkit import parse_variables
 from gpfit.fit_constraintset import FitCS
 from .sparloading import SparLoading
 
-#pylint: disable=invalid-name, no-member, arguments-differ
-#pylint: disable=attribute-defined-outside-init
+#pylint: disable=invalid-name, no-member, arguments-differ, exec-used
+#pylint: disable=attribute-defined-outside-init, undefined-variable
 
 class GustL(SparLoading):
-    "spar loading model"
+    """ Gust Loading Model
+
+    Variables
+    ---------
+    vgust       10      [m/s]       gust velocity
+    Ww                  [lbf]       wing weight
+    v                   [m/s]       vehicle speed
+    cl                  [-]         wing lift coefficient
+
+    Variables of length wing.N
+    --------------------------
+    agust                           [-]         gust angle of attack
+    cosminus1   self.return_cosm1   [-]         1 minus cosine factor
+
+    Upper Unbounded
+    ---------------
+    v, cl
+
+    Lower Unbounded
+    ---------------
+    Ww
+
+    LaTex Strings
+    -------------
+    vgust               V_{\\mathrm{gust}}
+    Ww                  W_{\\mathrm{w}}
+    cl                  c_l
+    agust               \\alpha_{\\mathrm{gust}}
+    cosminus1           (cos(x)-1)
+
+    """
     new_qbarFun = None
     new_SbarFun = None
 
-    def setup(self, wing):
+    return_cosm1 = lambda self, c: hstack(
+        [1e-10, 1-cos(c[self.wing.planform.eta][1:]*pi/2)])
```

Comment:
this makes me wonder if it should be an N-1 sized variable, skipping that first `eps` entry